packages/react-experiments/src/components/SelectedItemsList/SelectedItemsList.tsx

```diff
@@ -31,6 +39,13 @@ const _SelectedItemsList = <TItem extends BaseSelectedItem>(
     });
     setItems(updatedItems);
     props.onItemsRemoved?.(itemsToRemove);
+
+    if (itemsToRemove.length === 1) {
+      const item = itemsToRemove[0];
+      onAnnounceDeletedRecipients?.(item.text);
```

Comment:
>Recipients [](http://example.com/codeflow?start=23&length=10)

Don't name this "recipients". The collection could be any number of "things" so we should keep this generic.